src/Context/AuthContext.tsx

```diff
@@ -0,0 +1,41 @@
+import { createContext, useEffect, useState } from "react";
+import { jwtDecode } from "jwt-decode";
+
+
+
+export  const AuthContext = createContext ({});
+
+
+export default function AuthContextProvider(props:any){
+
+  const [userData, setUserData] = useState(null);
+  const[role,setRole]=useState(null)
+  
+
+  let reqHeaders={
+    Authorization:`Bearer ${localStorage.getItem("userToken")}`
```

Comment:
You 'll need to remove `Bearer` here as the token coming from the backend is `Bearer TokenValue`, so if you used the current token the request header will be `Bearer Bearer TokenValue`

![image](https://github.com/AhmedEl-Hussin/Hotel-System/assets/64327685/f736454a-f367-4024-8076-77b13b00574a)